modules/wordpress-com-block-editor-iframe.php

```diff
@@ -111,3 +111,64 @@ function jetpack_filter_salt( $salt, $scheme ) {
 
 	return $salt;
 }
+
+/**
+ * Enqueue the scripts for the WordPress.com block editor integration.
+ */
+function jetpack_enqueue_wpcom_block_editor_scripts() {
+	$suffix        = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+	$version       = gmdate( 'YW' );
+	$is_calypsoify = 1 === (int) get_user_meta( get_current_user_id(), 'calypsoify', true );
+	wp_enqueue_script(
+		'wpcom-block-editor-common',
+		'//widgets.wp.com/wpcom-block-editor/common' . $suffix . '.js',
```

Comment:
this might hurt our ability to find the script in the future. we might consider creating a bit of duplication in order to preserve searchability

```php
$use_debug = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG;
$path = $use_debug
	? '//widgets.wp.com/wpcom-block-editor/common.min.js'
	:  '//widgets.wp.com/wpcom-block-editor/common.js';
```